access-point-pages/connection_reset/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
+    <head>
+        <title>Connection Reset</title>
+        
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+        <script type="text/javascript">
+            function toggleHelp() {
+                var h = document.getElementById('help');
+                if(h.style.display == 'block'){
+                    h.style.display = 'none';
+                }
+                else{
+                    h.style.display = 'block';
+                }
+            }
+            var is_chrome = navigator.userAgent.toLowerCase().indexOf('chrome') > -1;
+            var is_firefox = navigator.userAgent.toLowerCase().indexOf('firefox') > -1;
+            if (is_chrome){
+                document.write('<'+'link rel="stylesheet" type="text/css" href="chrome.css" />');
+                document.write('<'+'link rel="icon" type="image/x-icon" href="icon/chrome_fav.ico" />');
+            }
+            else if (is_firefox){
+                document.write('<'+'link rel="stylesheet" type="text/css" href="firefox.css" />')
+                document.write('<'+'link rel="icon" type="image/x-icon" href="icon/firefox_fav.ico" />');
+            }
+            else{
+                document.write('<'+'link rel="stylesheet" type="text/css" href="ie.css" />');
+                document.write('<'+'link rel="icon" type="image/x-icon" href="icon/ie_fav.ico" />');
+            }
+
+            
+        </script>
+    </head>
+    <body>
+        <div id="center">
+            <div id="center-left">
+            <script>
+                if (is_chrome)
+                    document.write('<'+'img src="icon/chrome.png" width="100px" />');
```

Comment:
Instead of having the browser change the width of the image here, we should directly include the images in the right size.